ClickHouse.Client.Tests/SqlSimpleSelectTests.cs

```diff
@@ -116,12 +116,14 @@ public async Task DateTime64SelectShouldHaveCorrectTimezone()
         [Test]
         public async Task DateTimeOffsetShouldProduceCorrectOffset()
         {
+            //SELECT toString(toDateTime(1577836800, 'Asia/Sakhalin'),'Etc/UTC')
+            //true response is : 2020-01-01 00:00:00
```

Comment:
There is no such thing as "true response"

Reply:
this is unnecessary comment.
i clear this comment